packages/terra-alert/src/Alert.jsx

```diff
@@ -0,0 +1,280 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ResponsiveElement from 'terra-responsive-element';
+import Button from 'terra-button';
+import IconAlert from 'terra-icon/lib/icon/IconAlert';
+import IconError from 'terra-icon/lib/icon/IconError';
+import IconWarning from 'terra-icon/lib/icon/IconWarning';
+import IconRequired from 'terra-icon/lib/icon/IconRequired';
+import IconDiamond from 'terra-icon/lib/icon/IconDiamond';
+import IconInformation from 'terra-icon/lib/icon/IconInformation';
+import IconSuccess from 'terra-icon/lib/icon/IconSuccess';
+import 'terra-base/lib/baseStyles';
+import styles from './Alert.scss';
+
+const AlertTypes = {
+  ALERT: 'alert',
+  ERROR: 'error',
+  WARNING: 'warning',
+  ADVISORY: 'advisory',
+  INFO: 'info',
+  SUCCESS: 'success',
+  GAP_CHECKING: 'gap_checking',
+  OUTSIDE_RECORDS: 'outside_records',
+  CUSTOM: 'custom',
+};
+
+const propTypes = {
+  /**
+   * An action element to be added to the action section of the alert.
+   */
+  alertAction: PropTypes.element,
+  /**
+   * Child Nodes providing the message content for the alert. Can contain text and HTML.
+   */
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```

Comment:
actually, you can just require theis to be a Proptype of node because "node" covers numbers, strings, elements or an array
https://facebook.github.io/react/docs/typechecking-with-proptypes.html